source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +22,7 @@ ReferenceCounter: class {
 			}
 		}
 	}
+	count ::= this _count
```

Comment:
Oh that's true, it is so only for properties with empty get/set like `isSafe: Bool {get set}`